### DISCUSSION TITLE
Do you like discussions? or prefer discord?

### DISCUSSION THREAD

**Discussion by mbround18** [mbround18/valheim-docker#104] [Ideas]
Do you like these discussions? are they easier? Or would you prefer a discord server?

**Comment by AtroposOrbis** [comment]
I really don't mind either. At least here everything is much more addressable and more easily maintained by those actively developing. If I had to choose, it'd be to stick with github to keep all documentation centralized.

**Reply by TheDakk** [reply]
I agree, keep it here for informational purposes. All the feedback discussions so far have been very helpful too!

**Comment by mbround18** [comment]
Using the new webhook feature, I created a discord to publish when latest or another version is built and released. To hold true to yalls feedback AtroposOrbis and TheDakk which I greatly appreciate <3 there are no chat permissions. But you can use the release notification channel to get updates to your discord server using Discords announcement feature. 

https://discord.gg/3kTNUZz276

Tbh, I got tired of checking github for when a build completed and I could pull the latest image build :P now it pings my phone and I can run the updates for testing.

**Comment by Maistho** [comment]
Discussions is a lot better for searchability imo. Very difficult to find old conversations in Discord, comparatively.
